include/kernels/MFEMLinearElasticityKernel.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include "MFEMKernel.h"
+
+/*
+(σ ∇ q, ∇ q')
```

Comment:
Worth correcting this short comment here representing the weak form of this kernel

Reply:
Oops that slipped through, I had a go at representing it but let me know if you want any changes

Reply:
Possibly the clearest form is in the tensor form written at `https://mfem.org/bilininteg/` under "Vector Field Operators" "Square Operators". 

As a bonus - it may be useful to include how the Lame coefficients can be calculated from knowledge of the Young's modulus and Poisson's ratio in this case - i.e. equations 11-13a and 11-13b from [here](https://cns.gatech.edu/~predrag/courses/PHYS-4421-04/lautrup/book/elastic.pdf)